node-feature-discovery-job.json.template

```diff
@@ -17,6 +17,14 @@
         }
       },
       "spec": {
+        "volumes": [
```

Comment:
Supplying `--net=host` should be sufficient. No need for additional bind-mounts or `--privileged`. See below.

```
root@d1k8s:~/sriov-demo/docker-iperf3-dpdk# ls /sys/class/net/
docker0  eth0  eth1  flannel.1  lo  veth8b16ae5  vethb3234d4
root@d1k8s:~/sriov-demo/docker-iperf3-dpdk# docker run -t debian bash -c "ls /sys/class/net"
eth0  lo
root@d1k8s:~/sriov-demo/docker-iperf3-dpdk# docker run -t --net=host debian bash -c "ls /sys/class/net"               
docker0  eth0  eth1  flannel.1  lo  veth8b16ae5  vethb3234d4
```